### PR TITLE
fixing cluster dashboard description

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/using-opscenter.adoc [hunk 1]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in, and the application deployment name.
+* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that all pods.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that of all pods. 
```

File: modules/ROOT/pages/using-opscenter.adoc [hunk 2]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in, and the application deployment name.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
** Filters include the environment ID the application is deployed in and the application deployment name.
```

File: modules/ROOT/pages/using-opscenter.adoc (hunk 1 above)
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]: Suggestion:
his dashboard shows only machine-level data for all pods.
or
This dashboard does not show ingress-specific data but it shows machine-level data for all pods.

File: modules/ROOT/pages/using-opscenter.adoc [hunk 3]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
...
+* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that all pods. 
+** Filters include `nodename`, allowing you to filter by a specific VM.
+* *CPU Core Usage* shows CPU usage for Mule applications only.
+* *Ingress* shows metrics on all the replicas running the internal load balancer.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
* *Ingress* shows metrics on all replicas running the internal load balancer.
```

File: modules/ROOT/pages/using-opscenter.adoc [hunk 4]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
...
+* *Pods*  shows metrics on each individual service running within the Runtime Fabric cluster. 
+** Filters include the `namespace` and `podname`, allowing you to specify the specific service to monitor.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc:68]:
```suggestion
** Filters include `namespace` and `podname`, allowing you to specify the specific service to monitor.
```

Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]: I don't think you should be enclosing namespace and pod name in backticks here.

File: modules/ROOT/pages/using-opscenter.adoc [hunk 5]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
...
+** When selecting `namespace`, you'll see GUIDs which correspond to your Anypoint environment IDs under
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
** When selecting `namespace`, you'll see GUIDs that correspond to your Anypoint environment IDs under
```

File: modules/ROOT/pages/using-opscenter.adoc (hunk 2 above)
Comment (dhirsch1) [reply modules/ROOT/pages/using-opscenter.adoc]: Suggestion: Filters include the ID for the environment where the application is deployed and the application deployment name.

File: modules/ROOT/pages/using-opscenter.adoc (hunk 5 above)
Comment (dhirsch1) [reply modules/ROOT/pages/using-opscenter.adoc]: Can you make this text and text elsewhere present tense? 
Maybe - ** When selecting `namespace`, the xxx screen shows the GUIDs that correspond to the Anypoint environment IDs under which your applications are deployed.

File: modules/ROOT/pages/using-opscenter.adoc [hunk 6]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment.
```
Comment (dhirsch1) [inline modules/ROOT/pages/using-opscenter.adoc]: replicas of an application deployment instead of for an application deployment???

**Review by dhirsch1 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/using-opscenter.adoc (hunk 6 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
* *Applications* shows metrics for all replicas of application deployment. 
```

File: modules/ROOT/pages/using-opscenter.adoc [hunk 7]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in and the application deployment name.
+* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that all pods.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard does not show ingress-specific data, but it machine-level data for all pods. 
```

File: modules/ROOT/pages/using-opscenter.adoc [hunk 8]
```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
...
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in and the application deployment name.
+* *Cluster* shows metrics on the overall Runtime Fabric cluster and provides a machine-level overview of the Kubernetes cluster. The limit for a node is the sum of all limits for individual pods scheduled on it. Unbound pods, like ingress, are not included in the limit, as they are not limited. This dashboard will not show ingress-specific data but machine-level data only, including that all pods. 
+** Filters include `nodename`, allowing you to filter by a specific VM.
+* *CPU Core Usage* shows CPU usage for Mule applications only.
+* *Ingress* shows metrics on all the replicas running the internal load balancer.
+* *Pods*  shows metrics on each individual service running within the Runtime Fabric cluster. 
+** Filters include the `namespace` and `podname`, allowing you to specify the specific service to monitor.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/using-opscenter.adoc]:
```suggestion
** Filters include `namespace` and `podname`, allowing you to specify the specific service to monitor.
```

File: modules/ROOT/pages/using-opscenter.adoc (hunk 5 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/using-opscenter.adoc]: good call

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**